package.json

```diff
@@ -45,6 +45,8 @@
     "react-dom": "^18.3.1",
     "react-markdown": "^9.0.1",
     "react-useportal": "^1.0.14",
+    "remark-directive": "^3.0.0",
```

Comment:
question: why do we need to install `remark-directive` separately here?

Reply:
[165b808](https://github.com/electron/website/pull/655/commits/165b808a25752b20b80a4e6fa103e4e2321b7688)